.github/workflows/tap_integration_test.yml.disable

```diff
@@ -0,0 +1,31 @@
+name: Test GraphTally Protocol
+on:
+  push:
+    # branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up Docker cache
+        uses: docker/setup-buildx-action@v2
+        with:
+          install: true
+
+      - name: Run make setup
+        run: make setup
```

Comment:
```suggestion
        run: just setup
```

Reply:
And the rest of these directives. Did you not want to un-mark this as disabled and give it a run in CI?